tests/unit/test_args.py

```diff
@@ -758,6 +758,32 @@ def test_account_id_endpoint_mode_disabled_on_unsigned_request(self):
         config = self.call_get_client_args()['client_config']
         self.assertEqual(config.account_id_endpoint_mode, 'disabled')
 
+    def test_disable_host_prefix_injection_default_client_config(self):
+        input_config = Config()
+        client_args = self.call_get_client_args(client_config=input_config)
+        config = client_args["client_config"]
+        self.assertEqual(config.inject_host_prefix, True)
+
+    def test_disable_host_prefix_injection_config_store(self):
+        self.config_store.set_config_variable(
+            "disable_host_prefix_injection",
+            True,
+        )
+        config = self.call_get_client_args()['client_config']
+        self.assertEqual(config.inject_host_prefix, False)
+
+    def test_disable_host_prefix_injection_client_config_overrides_config_store(
```

Comment:
Should this name be `test_inject_host_prefix_client_config_overrides_disable_host_prefix` or `test_inject_host_prefix_client_config_overrides_config_store`? 

It's confusing to refer to `inject_host_prefix` as `disable_host_prefix_injection`.

Reply:
Fixed in fadc0255b080257fe358345bd2674bb18a75cebf